README.md

```diff
@@ -780,6 +780,8 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Omkar Kulkarni](https://github.com/Omkar0114)
   - [Onkar Dighe](https://github.com/onkardighe)
   - [Osoraa Nwankwo](https://github.com/Osoraa)
+  - [Omkar Ramishte](https://github.com/omk-coder)
```

Comment:
Please align yourself in alphabetical order also with the second, third and so on letter.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix